docs/architecture.md

```diff
@@ -42,10 +42,29 @@ The frontend will host the internal admin interface. The current application is
 
 ## Additional packages
 
+### HTTP Signature Utils
+
+The `http-signature-utils` package includes helper functions for
```

Comment:
something about this being for Open Payments client keys/signatures.

Reply:
And maybe a link to https://datatracker.ietf.org/doc/draft-ietf-httpbis-message-signatures/?